### DISCUSSION TITLE
Zeropages for assembler code

### DISCUSSION THREAD

**Discussion by adaumann** [drmortalwombat/oscar64#66] [General]
Currently I'm using a SID FX Sound tool to integrate sound effects (https://agpx.itch.io/sid-sfx-editor) Basically it works but in some cases I get some random effects. The tool generates a small byte code player I can execute, I can define 4 zeropage addresses for the player (default 252-255).I think the zeropages are in conflict somehow. Which 4 zeropage address could I use or can I reserve it?

**Comment by drmortalwombat** [comment]
`#pragma region( zeropage, 0x80, 0xf8, , , {zeropage})`

The compiler uses the BASIC range of zero page per default for its temporaries etc.  Zero page variables are allocated in the zeropage region.  You can redefine this region (see sample above)

**Reply by adaumann** [reply]
Works now, great!
